graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -355,17 +356,23 @@ c10::intrusive_ptr<FusedSampledSubgraph> FusedCSCSamplingGraph::InSubgraph(
 auto GetNumPickFn(
     const std::vector<int64_t>& fanouts, bool replace,
     const torch::optional<torch::Tensor>& type_per_edge,
-    const torch::optional<torch::Tensor>& probs_or_mask) {
+    const torch::optional<torch::Tensor>& probs_or_mask,
+    bool with_seed_offsets) {
```

Comment:
if we use a torch::optional<int64_t> for seed_offset, this boolean is not needed.